### DISCUSSION TITLE
Dirección Tests

### DISCUSSION THREAD

**Discussion by bozzimuc** [IIC2133-PUC/2024-2#55] [Tarea 0] [answered]
Buenas tardes, ¿dónde puedo encontrar los tests públicos para la Tarea 0? Muchas gracias

**Comment by sebacarrascop** [comment, marked as answer]
en el anuncio de canvas donde se anuncio la tarea

**Reply by bozzimuc** [reply]
Vale, ¡Muchas gracias!
